components/infoColumns/index.tsx

```diff
@@ -60,31 +59,32 @@ const CardContainer = styled.div<{
   ${breakpoints("margin", "", [{ 1400: "1rem 0" }])};
 `;
 const Title = styled(SubTitle)`
-  font-size: 4rem;
-  ${breakpoints("font-size", "", [{ 1000: "2.5rem" }])};
+  font-size: 6rem;
+  ${breakpoints("font-size", "", [{ 1000: "2.25rem" }])};
```

Comment:
If there's this large of a jump maybe have another breakpoint that splits the difference like 4rem @ 1200 ish, also if this gets sized down to 2.25rem make sure the other subtitles are less than or equal to 2.25rem to preserve the visual hierarchy